### ISSUE TITLE
Add support for crendentials

### ISSUE THREAD

**Issue by lpellegr** [ow2-proactive/studio#183]
Since the studio does not provide an authentication mode based on credentials (.cred files), it seems currently not possible to submit a task for execution as *runAsMe* from the studio when the property `pas.launcher.forkas.method` is set to `key`.

Currently, the workaround is to export the workflow, then to authenticate in the scheduler web portal or REST API using credentials before submitting.

**Comment by youribonnaffe** [comment]
Or to use third party credentials where the a special key can be used for the private SSH key

**Comment by lpellegr** [comment]
Yes, I saw a mention to this feature in the documentation and tried it but without success. Do you know if the content to set as value is an absolute path to the private key or the value of the private key ? I have tried the second solution only. It may explain why it was not working. The documentation is not clear regarding that point.

**Comment by youribonnaffe** [comment]
It should be the value of the private key. The keys to use is SSH_PRIVATE_KEY.

**Comment by lpellegr** [comment]
I performed a few tests again. *runAsMe* using SSH keys is working if the value associated to the `SSH_PRIVATE_KEY` for third party credentials is the value of the private key without any comment. 

For instance, the following comments starting by `-----` must be omitted : 

```
-----BEGIN RSA PRIVATE KEY-----
private_key_value
-----END RSA PRIVATE KEY-----
```

However, this option is complex to deploy since nodes must deployed with option `-Dpas.launcher.forkas.method=key`, SSH keys generated, SSH access configured, and specific third party credentials defined before using *runAsMe*. Having an uniform manner to authenticate between the studio and the web portal will improve the procedure a lot and make users happy.

**Comment by fviale** [comment]
fixed by 213e6b5
